src/app/organization/holidays/create-holiday/todo-flat-item.class.ts

```diff
@@ -0,0 +1,10 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class TodoItemFlatNode {
```

Comment:
Why is it called "Todo..."?

Reply:
Yes, that was not the correct naming. It is now renamed to office.